### DISCUSSION TITLE
how to install porteus distro on mob phone sd card via library central serviced ubuntu network

### DISCUSSION THREAD

**Discussion by domeniccaengmailcom** [uBlockOrigin/uAssets#24444] [General]
searching for most best way to do so either using the library terminal or my mob phone. the lib terminal wont accept any form of script - unable to install any packages and I'm no hacker. Any ideas

DC

**Comment by garry-ut99** [comment]
Wrong repository.

**Comment by domeniccaengmailcom** [comment]
thanks, ill try again

**Comment by domeniccaengmailcom** [comment]
not resolved, possibly redirected but not resolved.
